docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "116af785-a9b7-4eaf-8a86-b1eeb235da18",
+   "metadata": {},
+   "source": [
+    "# Fractional gates\n",
+    "\n",
+    "This page introduces two newly supported gate types on the IBM Quantum fleet of QPUs. These *fractional* gates are supported on:\n",
```

Comment:
This is not correct. Eagle must implement RZX and Heron must implement RZZ. Unfortunately we don't plan to provide fractional gates for Eagle, so in here you can just say fractional gates will be provided with Heron devices.